### ISSUE TITLE
imap4.py v1.131 typos

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7463]
|<img alt="anadelonbrin's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| anadelonbrin reported|
|-|-|
|Trac ID|trac#132|
|Type|defect|
|Created|2003-07-27 12:04:23Z|

<details><summary>Searchable metadata</summary>

```
trac-id__132 132
type__defect defect
reporter__anadelonbrin anadelonbrin
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1059307463000000 1059307463000000
changetime__1465000867693683 1465000867693683
version__ 
owner__ 
cc__exarkun cc__anadelonbrin
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="anadelonbrin's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|anadelonbrin commented|
|-|-|

```
#!html
<pre>
There are several instances of "k. v" in imap4.py where "k, 
v" is meant.  (all in consumeFetchResponse)

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Resolved, thanks

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph commented|
|-|-|

In changeset 408b19c0467f0746deec3b16ce9ae23088f8f2ae
```
#!CommitTicketReference repository="" revision="408b19c0467f0746deec3b16ce9ae23088f8f2ae"
Merge pull request #132 from alex/patch-1

Author: alex

Reviewer: glyph

Fixes: #8416

Fixed some markup to avoid ugly artifacts.
```
